cloud

```diff
@@ -1243,6 +1246,23 @@ function pubsub_withdraw() {
     pubsub_topic_withdraw "$project" "$pick_notifications_trigger_topic"
 }
 
+# Deploy a Google Cloud Storage Bucket
+# Args: project bucket
+function storage_deploy() {
+    declare -r project="$1"; shift
+    declare -r bucket="$1"; shift
+    TMPDIR="$TMPDIR_ORIG" gsutil -q mb -p "$project" -c STANDARD -l \
+        "us-central1" -b on "gs://$bucket"
```

Comment:
Don't edit this now, but it's good to wrap the option name *together* with the option value, because then the command is easier to read.

Reply:
Updated